DQM/GEM/plugins/GEMDQMHarvester.cc

```diff
@@ -0,0 +1,148 @@
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+//DQM services
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+
+#include <iostream>
+#include <cstdlib>
+#include <string>
+#include <memory>
+#include <vector>
+
+#include "TString.h"
+#include "Geometry/GEMGeometry/interface/GEMGeometry.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+
+using namespace std;
+using namespace edm;
+
+class GEMDQMHarvester: public DQMEDHarvester
+{  
+public:
+
+  GEMDQMHarvester(const edm::ParameterSet&);
+  ~GEMDQMHarvester() override;
+    
+protected:
+  void dqmEndJob(DQMStore::IBooker &, DQMStore::IGetter &) override {}
+  
+  void dqmEndLuminosityBlock(DQMStore::IBooker &, DQMStore::IGetter &, const edm::LuminosityBlock &, const edm::EventSetup &) override;
+
+  
+private:
+
+  void prova(DQMStore::IBooker &ibooker, edm::Run const &, edm::EventSetup const & iSetup);
+  std::string fName;
+  int verbosity;
+  DQMStore *dbe;
```

Comment:
please remove this pointer

Reply:
it was done. This moment we don't use harvester, so I removed all useless things.